lib/internal/http2.js

```diff
@@ -0,0 +1,812 @@
+'use strict';
+
+const http2 = process.binding('http2');
+const EventEmitter = require('events');
+const TLSServer = require('tls').Server;
+const NETServer = require('net').Server;
+const stream = require('stream');
+const Writable = stream.Writable;
+const Readable = stream.Readable;
+const PassThrough = stream.PassThrough;
+const constants = http2.constants;
+
+const kHandle = Symbol('handle');
+const kType = Symbol('type');
+const kStream = Symbol('stream');
+const kStreams = Symbol('streams');
+const kHeaders = Symbol('headers');
+const kHeadersSent = Symbol('headers-sent');
+const kSession = Symbol('session');
+const kOutgoingData = Symbol('outgoing-data');
+const kRequest = Symbol('request');
+const kResponse = Symbol('response');
+const kFinished = Symbol('finished');
+const kSocket = Symbol('socket');
+const kTrailers = Symbol('trailers');
+const kChunks = Symbol('chunks');
+const kProvider = Symbol('provider');
+const kPaused = Symbol('paused');
+const kResume = Symbol('resume');
+const kBeginResponse = Symbol('begin-response');
+const kEndStream = Symbol('end-stream');
+const kHasTrailers = Symbol('has-trailers');
+
+Object.defineProperty(exports, 'constants', {
+  configurable: false,
```

Comment:
`configurable: false` is not the default value ? In that case, we need to be explicit ?